bootstrapvz/base/bootstrapinfo.py

```diff
@@ -131,6 +138,12 @@ def filter_state(state):
 		state['__class__'] = self.__module__ + '.' + self.__class__.__name__
 		return state
 
+	def __parse_image_name(self, manifest, manifest_vars):
+		image_name = manifest.image['name'].format(**manifest_vars)
+		# Ensure that we do not use disallowed characters in image name
+		image_name = image_name.lower().replace('.', '-').replace(' ', '-')
```

Comment:
This is another reason why your suggestion of dealing with this at the manifest level would be better. :-)